unittests/forked_tests.cpp

```diff
@@ -333,19 +309,21 @@ BOOST_AUTO_TEST_CASE( prune_remove_branch ) try {
       c.push_block(fb);
    }
 
-   BOOST_REQUIRE_EQUAL(73u, c.control->head_block_num());
+   BOOST_REQUIRE_EQUAL(fork_num + 24u, c.control->head_block_num());
```

Comment:
Oh I completely missed that I had changed the behavior of the test! Thanks for noticing that.

Reply:
I fixed what we produce before `push_blocks`, so that we match the old version of the test. Now it does pick (and checks) the shorter fork. Thanks again for pointing my mistake. I hope my comment after `c.produce_block(); ` is accurate.

Reply:
Looks good!